Snakefile

```diff
@@ -240,6 +240,7 @@ rule build_bus_regions:
         country_shapes="resources/shapes/country_shapes.geojson",
         offshore_shapes="resources/shapes/offshore_shapes.geojson",
         base_network="networks/base.nc",
+        #gadm_shapes="resources/shapes/MAR2.geojson",
```

Comment:
Where applicable, instead of leaving this comment, in the same line of gadm_shapes below, may we add a comment on how to change these shapes with custom shapes? [This applies also for the following rules]
I was wondering also if the name "gadm" may be misleading in general. That may be especially true for cluster network, maybe cluster_shapes may be more appropriate? what do you think also?

Reply:
@Hazem-IEG Very good! May you add the direct link to the file to download? I think it would be way faster to use